src/Offer/Offer.php

```diff
@@ -19,20 +19,20 @@ final class Offer
     private $calendarType;
 
     /**
-     * @var DateTimeImmutable
+     * @var DateTimeImmutable|null
      */
     private $startDate;
 
     /**
-     * @var DateTimeImmutable
+     * @var DateTimeImmutable|null
```

Comment:
I think you forgot to make the getters for these nullable.

Reply:
Fixed that, good catch